src/main/java/com/coravy/hudson/plugins/github/GithubProjectProperty.java

```diff
@@ -21,9 +20,8 @@
  * As of now this is only the URL to the github project.
  *
  * @author Stefan Saasen <stefan@coravy.com>
- * @todo Should we store the GithubUrl instead of the String?
```

Comment:
Because the proposal is not a good one. Better to keep the XStream form as simple as possible, and where possible (not here, alas) aligned with the databinding form.